Doc/whatsnew/2.7.rst

```diff
@@ -2034,15 +2034,15 @@ version 1.3.  Some of the new features are:
   "xml" (the default), "html", or "text".  HTML mode will output empty
   elements as ``<empty></empty>`` instead of ``<empty/>``, and text
   mode will skip over elements and only output the text chunks.  If
-  you set the :attr:`tag` attribute of an element to ``None`` but
+  you set the :attr:`!tag` attribute of an element to ``None`` but
```

Comment:
Could link to https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.tag

```suggestion
  you set the :attr:`~xml.etree.ElementTree.Element.tag` attribute of an element to ``None`` but
```